afanasy/src/watch/listtasks.cpp

```diff
@@ -872,3 +858,20 @@ bool ListTasks::v_filesReceived( const af::MCTaskUp & i_taskup )
 	return true;
 }
 
+void ListTasks::getTaskInfo(int i_job_id, int i_block_num, int i_task_num, const std::string &i_mode, int i_number)
```

Comment:
I think that this should be a static method of `ItemJobTask` actually, since it concerns items and not lists.